src/utils/tribes.ts

```diff
@@ -112,6 +112,9 @@ async function initializeClient(
           sphinxLogger.info(`connected!`, logging.Tribes)
           if (!clients[username]) clients[username] = {}
           clients[username][host] = cl // ADD TO MAIN STATE
+          // subscribe here
+          await specialSubscribe(cl, contact)
```

Comment:
The only thing I can think of is this is in a scope withing cl.on and it should be outside?